traffic_monitor/towrap/towrap.go

```diff
@@ -352,7 +354,9 @@ func (s TrafficOpsSessionThreadsafe) CRConfigRaw(cdn string) ([]byte, error) {
 		b, reqInf, e := ss.GetCRConfig(cdn)
 		err = e
 		data = b
-		remoteAddr = reqInf.RemoteAddr.String()
+		if reqInf.RemoteAddr != nil {
+			remoteAddr = reqInf.RemoteAddr.String()
```

Comment:
This code looks good to me, but what happens if we don't provide a value for `remoteAddr` later on down the line? According to the GitHub issue this will likely fix that particular panic, but do we end up with a panic later since the `remoteAddr` is potentially nil? Should we give it an empty string or use `localHostIP` like it does later if there was an error?

Reply:
Well, `remoteAddr` is a string, so it cannot be `nil`. It will default to an empty string if no value is provided.